tests/update/prepare_system_for_update_tests.pm

```diff
@@ -24,6 +24,7 @@ sub run() {
     assert_screen('xterm-started');
     assert_script_sudo "chown $testapi::username /dev/$testapi::serialdev";
     assert_script_sudo "echo \"download.use_deltarpm = false\" >> /etc/zypp/zypp.conf";
+    assert_script_sudo "pkcon refresh";
```

Comment:
oh - and 'pkcon refresh' would not require su privileges - it queries the daemon through dbus